apps/data-importer/docs/testing.md

```diff
@@ -0,0 +1,9 @@
+## How to test Data importer app
```

Comment:
this app is actually an integration with the nuvo, but you don't mention it at all